modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -345,20 +345,18 @@ When you configure a node with a power saving configuration, you must configure
 By disabling P-states and C-states at the pod level, you can configure high priority workloads for best performance and lowest latency.
 
 .Configuration for high priority workloads
-[cols="1,2", options="header"]
-|====
-|Annotation
-|Description
+[cols="1,2,3", options="header"]
 
-a|[source,yaml]
-----
-annotations:
-  cpu-c-states.crio.io: "disable"
-  cpu-freq-governor.crio.io: "<governor>"
-----
-|Provides the best performance for a pod by disabling C-states and specifying the governor type for CPU scaling. The `performance` governor is recommended for high priority workloads.
-|====
+|===
+| Annotation | Possible Values | Description
+
+|`cpu-c-states.crio.io:` a|  * `enable`
+* `disable`
+* `max_latency:microseconds` | This annotation allows you to `enable` or `disable` C-states for each CPU. Alternatively, you can also specify a maximum latency in microseconds for the C-states. For example enable C-states with a maximum latency of 10 microseconds with the setting cpu-c-states.crio.io: `"max_latency:10"`. Set the value to `disable` to provide the best performance for a pod.
```

Comment:
```suggestion
* `max_latency:microseconds` | This annotation allows you to `enable` or `disable` C-states for each CPU. Alternatively, you can also specify a maximum latency in microseconds for the C-states. For example, enable C-states with a maximum latency of 10 microseconds with the setting `cpu-c-states.crio.io`: `"max_latency:10"`. Set the value to `disable` to provide the best performance for a pod.
```

Reply:
A couple of formatting suggestions here. And a question about why "max_latency:10" is in both quotes and backticks?

Reply:
My understanding is looking at this PR enable, disable and max_latency:10 need to be in "..." (quotes)

Reply:
Yes - the values all need to be in double quotes.